dev/sbt-plugin/src/sbt-test/sbt-plugin/projections-happy-path/hello/src/main/java/com/example/hello/impl/readsides/StartedProcessor.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example.hello.impl.readsides;
+
+import com.lightbend.lagom.javadsl.persistence.ReadSide;
+
+import javax.inject.Inject;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class StartedProcessor extends AbstractReadSideProcessor {
+    public static final String NAME = "StartedProcessor";
+
+    private static ConcurrentHashMap<String, String> greetings = new ConcurrentHashMap<>();
+    @Inject
+    public StartedProcessor(ReadSide readSide) {
+        super(NAME, readSide, greetings);
+    }
+}
```

Comment:
Instead of creating two fully-fledged read side processors backed by a database table I created a `AbstractReadSideProcessor` which gets an in-memory `ConcurrentHashMap` each event handler updates. Then, Icreate a `ConcurrentHashMap` for each specialization of the read side processor.